noxfile.py

```diff
@@ -184,7 +198,7 @@ def black(session):
 
 
 @nox.session(python=PY_VER, venv_backend="conda")
-def tests(session):
+def tests(session: nox.sessions.Session):
```

Comment:
Why is `session: nox.sessions.Session` not necessary with the subsequent sessions in this file?

Reply:
They are necessary. I hadn't completed the type annotations.  Done now